docs/pages/guides/authentication.mdx

```diff
@@ -171,6 +172,141 @@ export default function App() {
+import { Button } from 'react-native';
+/* @info Install the Beyond Identity Embedded SDK. This can only be used in standalone and bare workflow native apps. This SDK cannot be used in the Expo Go app. */
+import { Embedded } from '@beyondidentity/bi-sdk-react-native';
+/* @end */
+
+export default function App() {
+  // Endpoint
+  const discovery = useAutoDiscovery(`https://auth-${REGION}.beyondidentity.com/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/applications/${APPLICATION_ID}`);
+  // Request
+  const [request, response, promptAsync] = useAuthRequest(
+    {
+      clientId: `${CLIENT_ID}`,
+      scopes: ['openid'],
+      redirectUri: makeRedirectUri({
+        /* @info The URI <code>[scheme]://</code> to be used in bare and standalone. If undefined, the <code>scheme</code> property of your app.json or app.config.js will be used instead. */
+        scheme: 'your.app'
+        /* @end */
+      }),
+    },
+    discovery
+  );
+
+  React.useEffect(() => {
+    const authenticate = async (url) => {
+      // Display UI for user to select a passwordless passkey
+      const passkeys = await Embedded.getPasskeys();
+
+      if (await Embedded.isAuthenticateUrl(url)) {
+        // Pass url and a selected passkey ID into the Beyond Identity Embedded SDK authenticate function
+        /* @info Parse query parameters from the 'redirectUrl' for a 'code' and then exchange that code for an access token */
+        const { redirectUrl } = await Embedded.authenticate(url, passkeys[0].id);
+        /* @end */
+      }
+    };
+
+     /* @info The response does not need to be of type 'success' if it has a url. The state value is stored in a JWT in the url 'request' paramater and the url will be validated through the Beyond Identity Embedded SDK.  */
+     if (response?.url) {
+     /* @end */
+      authenticate(url)
+    }
+  }, [response]);
+
+  return (
+    <Button
+      /* @info Disable the button until the request is loaded asynchronously. */
+      disabled={!request}
+      /* @end */
+      title="Passwordless Login"
+      onPress={() => {
+        /* @info Prompt the user to authenticate in a user interaction or web browsers will block it. */
+        promptAsync();
+        /* @end */
+      }}
+    />
+  );
+}
+```
+</Tab>
+
+<Tab>
+Set your Beyond Identity [Authenticator Config][c-beyondidentity-authenticator-config]'s Invocation Type to <code>"Manual"</code>. If "Manual" is selected, an authentication URL be returned to you as part of a JSON response. No redirects are needed and will not require authentication through a web service. The result is a completley silent OAuth 2.0 authentication using Passkeys. Since the challenge is packaged as part of the authentication URL, following the URL will result in the same behavior as if an Invocation Type of "Automatic" were selected.
+{/* prettier-ignore */}
+```tsx Beyond Identity Auth Code w/ Invocation Type "Manual"
+import * as React from 'react';
```

Comment:
```suggestion
import { useEffect } from 'react';
```

Reply:
This one doesn't actually use `useEffect` so I'll change it to `import React from 'react';`

Reply:
Sounds good!